packages/web/pages/reel.tsx

```diff
@@ -0,0 +1,28 @@
+import { Step, Stepper, StepsIndicator } from "~/components/stepper";
+
+const Reel = () => {
+  return (
+    <div className="w-fit">
+      <Stepper autoplay={{ delay: 4000, stopOnHover: true }}>
+        <Step>
+          <h4>Heading</h4>
```

Comment:
https://www.figma.com/file/RxNOvfvzmgtsgCPUetQvcG/Supercharged-Liquidity?type=design&node-id=1173-47583&t=Xnm2aw28JnlyUuow-4

Check out this design needed for CL. Is there a way we will be able to hook into the timer and display a progress bar for each step, like seen with stories?